backend/corpus_asset_pipelines/integrated_corpus/transform.py

```diff
@@ -37,6 +38,32 @@ def apply_pre_concatenation_filters(
     return anndata_object
 
 
+def create_high_level_tissue(anndata_object: anndata.AnnData):
+    anndata_object.obs["tissue_original"] = anndata_object.obs["tissue"]
+    anndata_object.obs["tissue_original_ontology_term_id"] = anndata_object.obs["tissue_ontology_term_id"]
+    anndata_object.obs = get_high_level_tissue(anndata_object.obs)
+
+
+# TODO finalize this function
+def get_high_level_tissue(obs: DataFrame):
+
+    for i in range(len(obs)):
+
+        if "lung" in obs["tissue"][i]:
```

Comment:
If the file that will eventually be used to map tissues to high-level tissues is of the following format:
`Row: (original tissue, high level tissue)`
then I would create a temporary file mapping original lung tissues to lung and use that here.

Reply:
This is a placeholder will be replaced, see comment above